src/js/containers/aboutTheData/AgenciesContainer.jsx

```diff
@@ -210,62 +210,62 @@ const AgenciesContainer = ({
         ]);
 
     const handlePageChange = (page) => {
-        if (activeTab === 'details') {
-            updateDetailsPagination({ page, limit: detailsLimit });
+        if (activeTab === 'submissions') {
+            updateSubmissionsPagination({ page, limit: submissionsLimit });
         }
         else {
-            updateDatesPagination({ page, limit: detailsLimit });
+            updatePublicationsPagination({ page, limit: submissionsLimit });
         }
     };
 
     const handleLimitChange = (limit) => {
-        if (activeTab === 'details') {
-            updateDetailsPagination({ page: detailsPage, limit });
+        if (activeTab === 'submissions') {
+            updateSubmissionsPagination({ page: submissionsPage, limit });
         }
         else {
-            updateDatesPagination({ page: datesPage, limit });
+            updatePublicationsPagination({ page: publicationsPage, limit });
         }
     };
 
     return (
         <>
             <div className="table-container" ref={tableRef} onScroll={handleScroll}>
-                {activeTab === 'details' && (
+                {activeTab === 'submissions' && (
                     <Table
-                        rows={renderDetails(details)}
+                        rows={renderDetails(allSubmissions)}
                         classNames={`usda-table-w-grid ${verticalStickyClass} ${horizontalStickyClass}`}
                         columns={agenciesTableColumns[activeTab]}
                         updateSort={handleUpdateSort}
                         currentSort={{
-                            field: detailsSort[0],
-                            direction: detailsSort[1]
+                            field: submissionsSort[0],
+                            direction: submissionsSort[1]
                         }}
                         error={error}
                         loading={areDetailsLoading} />
                 )}
-                {activeTab === 'dates' && (
+                {activeTab === 'publications' && (
                     <Table
-                        rows={renderDates(dates)}
+                        rows={renderDates(allPublications)}
                         classNames={`usda-table-w-grid ${verticalStickyClass} ${horizontalStickyClass}`}
                         columns={agenciesTableColumns[activeTab]}
                         updateSort={handleUpdateSort}
                         currentSort={{
-                            field: datesSort[0],
-                            direction: datesSort[1]
+                            field: publicationsSort[0],
+                            direction: publicationsSort[1]
                         }}
                         error={error}
                         loading={areDatesLoading} />
                 )}
             </div>
-            {(details.length || dates.length) && (
+            {(allSubmissions.length || allPublications.length) && (
```

Comment:
Do we want to show pagination on error and loading?

Reply:
@jonhill13 https://github.com/fedspendingtransparency/usaspending-website/pull/2302/files?file-filters%5B%5D=.jsx#diff-5e7810fb3a893bd1fbfd8f908507a8a1ebd31de25d46717b5bb3273a992846d0R265